permissions/plugin-embotics-vcommander.yml

```diff
@@ -5,4 +5,7 @@ paths:
 developers:
 - "aelliotatemboticsdotcom"
 - "embotics"
-- "builderatemboticsdotcom"
+# Disabled by oleg-nenashev on Jun 15, 2020 - No valid email in Jira for password reset.
+# Please contact jenkins-infra@googlegroups.com to get the password reset.
+# See https://groups.google.com/forum/#!topic/jenkinsci-dev/3UvrCTflXGk for more details
+# - "builderatemboticsdotcom"
```

Comment:
Probably easy enough to fix, has the email address that exactly matches the user name in Artifactory.

Reply:
I updated this user email address